cycler.py

```diff
@@ -185,11 +185,11 @@ def change_key(self, old, new):
         if old == new:
             return
         if new in self._keys:
-            raise ValueError("Can't replace %s with %s, %s is already a key" %
-                             (old, new, new))
+            raise ValueError("Can't replace {} with {}, {} is already a key"
+                             .format(old, new, new))
```

Comment:
```suggestion
            raise ValueError(
                "Can't replace {old} with {new}, {new} is already a key"
                .format(old=old, new=new))
```

If we start changing formatting, please go with named placeholders. First, this removes the order dependence and the duplication (old, new, new). Second, this makes it trivial to switch to f-strings later on. (We cannot switch to f-strings immediately because we don't require 3.6+ yet, and I don't want to get into a version discussion right now).